src/codeu/chat/client/core/View.java

```diff
@@ -143,6 +136,49 @@ public View(ConnectionSource source) {
   }
 
   @Override
+  public Collection<UserAccessLevel> getAccessLevels(Uuid conversation) {
```

Comment:
This (and all of the associated `getAccessLevels()` methods and code that works with `GET_ALL_ACCESS_LEVELS_REQUEST` and `GET_ALL_ACCESS_LEVELS_RESPONSE`) can't actually be called, can it?

I'm guessing that you all were planning to add a user-visible command in the Conversation context that would dump the list of everybody's access levels for this conversation. What happened?

It's not required for the project (unless I missed something?), so my suggestion would be to just delete this partway-implemented command.